kola-denylist.yaml

```diff
@@ -31,3 +35,13 @@
 # but not denylisted here so it can run on the rhcos pipeline
 #- pattern: iso-offline-install-iscsi.ibft.bios
 #  tracker: https://github.com/openshift/os/issues/1492
+
+# as it's a fake rhel build (from c9s) versions won't match
+- pattern: ext.config.version.rhel-major-version
+  tracker: https://github.com/openshift/os/issues/1635
+  osversion:
+    - rhel-9.6
+- pattern: ext.config.shared.content-origins
+  tracker: https://github.com/openshift/os/issues/1635
+  osversion:
+    - rhel-9.6
```

Comment:
can we add a snooze to these so that we don't forget to start running the test?

I wish we could do it for all of the UEFI ones above but those have `c9s` on them too.